src/main/java/seedu/address/storage/JsonAdaptedTask.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.animal.Task;
+import seedu.address.model.animal.TaskList;
+
+
+
+/**
+ * A Jackson-friendly adapted version of the {@link TaskList} model class.
+ */
+public class JsonAdaptedTask {
+    private final String description;
+    private final String taskStatus;
```

Comment:
Consider serializing `taskStatus`, or `isDone` in this case as-is, as a boolean value, to avoid having to do a check `taskStatus.equals("X")` to determine whether to deserialize as true or false.

Reply:
Will do so as I work on the storage!